site/learn/BestPractice-Authorization.md

```diff
@@ -3,6 +3,56 @@ title: Authorization
 layout: ../_core/DocsLayout
 category: Best Practices
 permalink: /learn/authorization/
+next: /learn/serving-over-http/
 ---
 
-WIP
+> Delegate authorization logic to the business logic layer
+
+Authorization is a type of business logic that describes whether a given user/session/context has permission to perform an action or see a piece of data. For example:
+
+*“Only authors can see their drafts”*
+
+Enforcing this kind of behavior should happen in the [business logic layer](/learn/thinking-in-graphs/#business-logic-layer). It is tempting to place authorization logic in the GraphQL layer like so:
+
+```javascript
+var postType = new GraphQLObjectType({
+  name: ‘Post’,
+  fields: {
+    body: {
+      type: GraphQLString,
+      resolve: function (post, args, context, rootValue) {
```

Comment:
Many popular projects are still documented without arrow functions. Express is a good example - https://expressjs.com/en/guide/routing.html So is Lodash - https://lodash.com/docs/4.15.0#flatMap - I agree that most GraphQL docs nowadays tend to assume you already know how to set up transpilation, but I think that is actually slowing adoption. Promises are the only case where I would suggest using arrow functions, because they are just too verbose with the `function` keyword